core/metacling/src/TCling.cxx

```diff
@@ -1093,7 +1093,11 @@ static GlobalModuleIndex *loadGlobalModuleIndex(cling::Interpreter &interp)
    // StringRef ModuleIndexPath = HSI.getModuleCachePath();
    // HeaderSearch& HSI = PP.getHeaderSearchInfo();
    // HSI.setModuleCachePath(TROOT::GetLibDir().Data());
+#ifdef R__WIN32
+   std::string ModuleIndexPath = TROOT::GetBinDir().Data();
```

Comment:
Shouldn't we be fixing the `GetLibDir` instead to work on Windows? Maybe we should provide a new interface like `TROOT::GetSharedLibDir` which captures this intent here. For non-windows it would forward to ``GetLibDir` otherwise GetBinDir` .

Reply:
Well, right, we can always find another solution. Do you want me to add yet another method? On Windows, the `bin` directory is where the executables are (a DLL is an executable), and the `lib` directory is where the libraries are. Do you really want to add a `GetSharedLibDir` method?